src/net_processing.cpp

```diff
@@ -2924,6 +2924,14 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             return;
         }
 
+        // If we are in IBD, exit from unrequested transaction processing early to save bandwidth
+        // and mitigate potential DoS risks.
```

Comment:
This doesn't save bandwidth since the tx is already downloaded. I think "Do not process unrequested transactions during IBD" is sufficient as a comment.

Reply:
There is a edge case where the transaction is a orphan and thus trigger further download but that's not worthy to document as bandwidth saving. Updated.